crossplane.yaml

```diff
@@ -34,65 +33,11 @@ metadata:
 spec:
   crossplane:
-    version: ">=v1.0.0-0"
+    version: ">=v1.7.0-0"
   dependsOn:
-    - provider: registry.upbound.io/crossplane/provider-gcp
-      version: ">=v0.18.0-0"
-    - provider: registry.upbound.io/crossplane/provider-helm
-      version: ">=v0.3.6-0"
+    - provider: xpkg.upbound.io/upbound/provider-gcp
+      version: ">=v0.13.0-0"
+    - provider: xpkg.upbound.io/crossplane/provider-helm
+      version: ">=v0.10.0-0"
```

Comment:
```suggestion
      version: ">=v0.11.1-0"
```

Reply:
@muvaf , thanks, updated!

During the configuration testing, I got v0.12.rc build installed. Is that expected?
```
 k get providers
NAME                       INSTALLED   HEALTHY   PACKAGE                                                            AGE
provider-aws               True        True      xpkg.upbound.io/upbound/provider-aws:v0.14.0                       5d15h
provider-gcp               True        True      xpkg.upbound.io/upbound/provider-gcp:v0.13.0                       16h
crossplane-provider-helm   True        True      xpkg.upbound.io/crossplane/provider-helm:v0.12.0-rc.0.5.gb368192   30s
```